TestMain.py

```diff
@@ -10,8 +10,12 @@ class TestMain(unittest.TestCase):
     def setUpClass(cls):
         # cloning the specimin 
         main.clone_repository('https://github.com/kelloggm/specimin.git', 'resources')
-        cls.json_data = main.read_json_from_file('resources/test_data.json')[0]
-        cls.specimin_dir = "resources/specimin"
+        cls.json_data = main.read_json_from_file('resources/test_data.json')[6]
+        sp_env_var = main.get_specimin_env_var()
+        if (sp_env_var): 
+            cls.specimin_dir = "resources/specimin"
```

Comment:
This looks like it sets `cls.specimin_dir` to `"resources/specimin" even if the environment variable is set?